Parse/src/main/java/com/parse/Parse.java

```diff
@@ -35,6 +36,86 @@
  * library.
  */
 public class Parse {
+  public static final class Configuration {
+    private final Context context;
+    private final String applicationId;
+    private final String clientKey;
+    private final boolean localDataStoreEnabled;
+    private final List<ParseNetworkInterceptor> interceptors;
+
+    private Configuration(Builder builder) {
+      this.context = builder.context;
+      this.applicationId = builder.applicationId;
+      this.clientKey = builder.clientKey;
+      this.localDataStoreEnabled = builder.localDataStoreEnabled;
+      this.interceptors = builder.interceptors;
+    }
+
+    public static final class Builder {
+      private Context context;
+      private String applicationId;
+      private String clientKey;
+      private boolean localDataStoreEnabled;
+      private List<ParseNetworkInterceptor> interceptors = new ArrayList<>();
+
+      public Builder(Context context) {
+        this.context = context;
+      }
```

Comment:
The problem is that `android.content.Context` doesn't exist without the Android APIs, which would mean this method would cause this class to fail to load in a pure Java environment due to `import android.content.Context` that's required.

However, this is already a problem with `com.parse.Parse` anyway, so this isn't really adding any additional problems since we'd need to either create a new class `com.parse.ParseJava` or break `com.parse.Parse` anyway.